project.clj

```diff
@@ -20,15 +21,19 @@
             [lein-cljsbuild "1.1.7" :exclusions [org.clojure/clojure
                                                  org.clojure/clojurescript]]
             [lein-figwheel "0.5.14" :exclusions [org.clojure/clojure
-                                                 org.clojure/clojurescript]]]
+                                                 org.clojure/clojurescript]]
+            [com.github.clj-kondo/lein-clj-kondo "0.2.4" :exclusions [org.clojure/clojure
```

Comment:
I’ve never used a plugin for this. I’ve just seen the main function called directly. Interesting.

Reply:
yeah, I've also only used the main function directly, but since `clara-rules` still uses `leiningen` this was the easiest way to get it setup for CI.